modules/apps/app-builder/app-builder-test/src/testFunctional/paths/FormViewBuilder.path

```diff
@@ -192,7 +192,7 @@
 <!--DATE-->
 <tr>
 	<td>DATE_FIELD</td>
-	<td>//p[contains(text(),'${key_fieldName}')]/..//input[1]</td>
+	<td>//p[contains(text(),'${key_fieldName}')]/../following-sibling::*//input[@aria-label='date_picker']</td>
```

Comment:
hey @matuzalemsteles, just fyi... I've updated the selector for Select from List and Date field. We have discussed recently and I noticed since week the CRUD App Builder tests began to fail... I'm fixing them on this commit. Thanks!

Reply:
Thanks @alexcandidos